app/helpers/samples_helper.rb

```diff
@@ -123,12 +123,11 @@ def samples_pipeline_run_info(samples)
   end
 
   def filter_samples(samples, query)
-    if query == 'UPLOADING'
-      samples = samples.where(status: 'created')
-      samples = samples.where(status: 'created')
-    else
-      samples = samples.joins("INNER JOIN pipeline_runs ON pipeline_runs.sample_id = samples.id").where(status: 'checked').where("pipeline_runs.id in (select max(id) from pipeline_runs group by sample_id)").where("pipeline_runs.job_status = '#{query}'")
-    end
+    samples = if query == 'UPLOADING'
+                samples.where(status: 'created')
+              else
+                samples.joins("INNER JOIN pipeline_runs ON pipeline_runs.sample_id = samples.id").where(status: 'checked').where("pipeline_runs.id in (select max(id) from pipeline_runs group by sample_id)").where("pipeline_runs.job_status = '#{query}'")
```

Comment:
You probably want to escape the query param, as oppose to passing the value directly into the sql, to avoid sql injection